strawberryfields/apps/graph/utils.py

```diff
@@ -151,13 +151,24 @@ def is_subgraph(subgraph: Iterable, graph: nx.Graph):
 
 
 def is_clique(graph: nx.Graph) -> bool:
-    """Determines if the input graph is a clique. A clique of n nodes has :math:`n*(n-1)/2` edges.
+    """Determines if the input graph is a clique. A clique of :math:`n` nodes has
+     :math:`n*(n-1)/2` edges.
```

Comment:
```suggestion
     :math:`n(n-1)/2` edges.
```
Since it renders like latex, the asterisk is probably not required